README.md

```diff
@@ -43,7 +43,15 @@ docker-compose down --remove-orphans
 
 ## Logging in
 
-Auth0 is required for login. You also need to add your client ID and secret to the env file in the Auth0 section.
+Auth0 or Devise can be used for login. For Auth0, you need to add your client ID and secret to the env file in the Auth0 section.
+
+## Caching
+
+You can view keys in the Redis cache from the console with:
+
+```
+Redis.new(url: "redis://redis:#{ENV['REDIS_PORT']}/0", password: ENV['REDIS_PASSWORD'], namespace: ENV['REDIS_CACHE_NAMESPACE']).keys('*')
+```
 
```

Comment:
how about just Rails.cache it won't have access to the keys ? like:
``` Rails.cache.instance_variable_get(:@data).keys```

Reply:
Can also be useful to be able to check Redis directly when debugging, like I didn't understand that it was using MemoryCache instead of RedisCache before, and that command helped me understand what was actually happening. But either works for README.

Reply:
Added both. Thanks for review!